src/Spago/Build.hs

```diff
@@ -141,8 +141,12 @@ runWithNode defaultModuleName maybeSuccessMessage failureMessage maybeModuleName
   where
     moduleName = fromMaybe defaultModuleName maybeModuleName
     args = Text.intercalate " " $ map Purs.unExtraArg nodeArgs
-    cmd = "node -e \"require('./output/" <> Purs.unModuleName moduleName <> "').main()\" " <> args
+    contents = "#!/usr/bin/env node\n\n" <> "require('../output/" <> Purs.unModuleName moduleName <> "').main()"
+    cmd = "node .spago/run.js " <> args
     nodeAction = do
+      echoDebug $ "Writing .spago/run.js"
```

Comment:
This will affect also `spago test`. We might not want to write a script in that case, or write a script with a different name

Reply:
hyi herranjestas